pyproject.toml

```diff
@@ -1,72 +1,67 @@
 [build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
 
 
-[tool.poetry]
+[project]
 name = "mainpy"
-version = "1.4.0"
 description = "Simplify your project entrypoint with @main"
-authors = ["Joren Hammudoglu <jhammudoglu@gmail.com>"]
-license = "MIT"
+version = "1.4.0"
+authors = [
+  {name = "Joren Hammudoglu", email = "jhammudoglu@gmail.com"},
+]
+license = {text = "MIT"}
 readme = "README.md"
-repository = "https://github.com/jorenham/mainpy"
-documentation = "https://github.com/jorenham/mainpy?tab=readme-ov-file#mainpy"
 classifiers = [
-    "Development Status :: 5 - Production/Stable",
-    "Operating System :: OS Independent",
-    "Typing :: Typed",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Programming Language :: Python :: 3.13",
+  "Development Status :: 5 - Production/Stable",
+  "Operating System :: OS Independent",
+  "Typing :: Typed",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
```

Comment:
these were automatic with poetry, I wonder if it's supported in `uv`

Reply:
UV's own [`pyproject.toml`](https://github.com/astral-sh/uv/blob/main/pyproject.toml) has them, so I suppose they should still be added manually